11_rtc_driver/.gitignore

```diff
@@ -0,0 +1,159 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
```

Comment:
Is this file and all patterns defined here are required for developed module. Please remove it or leave only patterns for intermediate files.